src/popup/components/Header.vue

```diff
@@ -198,12 +198,15 @@ export default defineComponent({
     display: flex;
     justify-content: flex-end;
     gap: 8px;
-    width: 20%;
+
+    .btn-close {
+      position: relative;
+      top: unset;
+      right: unset;
+    }
```

Comment:
I think this styles should replace the styles used in the `BtnClose`. From my experience components should not position themselves.